flink-yarn-tests/src/test/java/org/apache/flink/yarn/YarnTestBase.java

```diff
@@ -621,13 +632,16 @@ protected static void sendOutput() {
 
 	public static class Runner extends Thread {
 		private final String[] args;
+		private final int expectedReturnValue;
 		private int returnValue;
```

Comment:
You can remove this field now and convert it into a local variable in the `run()` method.